vector/src/main/java/im/vector/app/features/home/room/detail/timeline/format/NoticeEventFormatter.kt

```diff
@@ -56,6 +57,27 @@ class NoticeEventFormatter @Inject constructor(private val activeSessionDataSour
 
     private fun Event.isSentByCurrentUser() = senderId != null && senderId == currentUserId
 
+    private fun isDirectRoom(roomId: String?): Boolean {
+        return roomId?.let {
+            activeSessionDataSource.currentValue?.orNull()?.getRoomSummary(roomId)?.isDirect
+        }.orFalse()
+    }
+
+    private fun chooseResourceByRoomType(event: Event,
+                                         @StringRes directRoomResId: Int,
+                                         @StringRes directRoomByUserResId: Int,
+                                         @StringRes publicRoomResId: Int,
+                                         @StringRes publicRoomByUserResId: Int,
```

Comment:
Is not really for public rooms but for not DM rooms. I do not know what to propose though...